report/verification/mms/radioactive_decay.md

```diff
@@ -0,0 +1,209 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.2
+kernelspec:
+  display_name: vv-festim-report-env
+  language: python
+  name: python3
+---
+
++++ {"vscode": {"languageId": "javascript"}}
+
+# Radioactive decay
```

Comment:
The two cases have the same name. Either indent them to another level or give them different names

Reply:
Indenting them to another level isn't possible because of how the ToC is structured, sadly. I've changed their names for now but we could pull the cases up a level, placing them all below methods, which would allow us to make sections for topics like radioactive decay. The other solution is the tags method I'd mentioned, which I'm looking into, there's a promising tool for it.